chandra_aca/dark_scale.py

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Do we need a new `dark_scale.py` module?  This one function fits in nicely with `dark_model` functions.